api/src/bin/main.rs

```diff
@@ -16,7 +17,14 @@ async fn main() {
 
     let harbor = app(Config::new(cx)).await;
 
-    let addr = SocketAddr::from(([0, 0, 0, 0], 5000));
+    // Retrieve the port number from the environment
+    // variable, defaulting to 5000 if it isn't set.
+    let port: u16 = env::var("PORT")
```

Comment:
Check out this file: `sdk/core/src/config.rs`.  This is how we are extracting environment variables.